build.gradle

```diff
@@ -62,21 +62,20 @@ ext {
   minSdkVersion = 21
   buildToolsVersion = "29.0.2"
   androidxAppCompat = "1.2.0"
-  androidxAnnotations = "1.1.0"
-  androidXLegacy = "1.0.0"
+  androidxAnnotations = "1.2.0"
   androidMaterial = "1.2.1"
   cardView = "1.0.0"
-  googleMaps = "17.0.0"
-  constraintLayout = "1.1.3"
+  googleMaps = "17.0.1"
+  constraintLayout = "2.0.4"
```

Comment:
this surprisingly went without any issues (thanks screenshot testing!)

Reply:
It may create issues in the app if they use our dependency transitively. Let's check it with app version.

Reply:
they're already on 2.0.4 as well